src/routers/secure/scrape.py

```
@@ -1,3 +1,4 @@
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)